README.md

```diff
@@ -105,9 +115,13 @@ Please see the [BLOCKv Android Example](https://github.com/BLOCKvIO/android-samp
 
 We recommend you use [Dagger 2](https://github.com/google/dagger), or a similar library, for singleton management for the BLOCKv SDK.
 
+## Production Support
+
+The BLOCKv SDK for Android is currently in public **beta**. Breaking changes may still be introduced in the coming months.  This is *important* to consider if you are planning on releasing the SDK as part of your application.
+
 ## Versioning
 
-The BLOCKv SDK for Android is still in pre-release and so may introduce breaking changes. Once the release is official, the SDK will follow [semantic versioning](https://semver.org), starting with release 1.0.0.
+Once the release is official, the SDK will adhere to [semantic versioning](https://semver.org).
 
 ## Author
 
```

Comment:
Did you update the title? "BLOCKv SDK for Android (BETA)"

Reply:
Missed that, thanks